src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -184,20 +184,17 @@ public void storeUserTeaData(
       datastoreUser.setProperty("username", datastoreUser.getProperty("username"));
       datastoreUser.setProperty("date", datastoreUser.getProperty("date"));
 
-      Map<String, Long> newMap = new HashMap<String, Long>();
       EmbeddedEntity teaMap = (EmbeddedEntity) datastoreUser.getProperty("teaData");
```

Comment:
This is valid, but I should look into this when I have the time.
https://github.com/objectify/objectify/wiki/Entities